examples/sklearn-sentiment-clf/sklearn-sentiment-clf.ipynb

```diff
@@ -277,12 +277,25 @@
     "!bentoml serve ./model"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Use following command to make an request to Rest Server for Prediction*"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "!curl -i \\\n",
+    "--header \"Content-Type: application/json\" \\\n",
+    "--method POST \\\n",
+    "--data '{\"text\": \"some new text, sweet noodles\"}'\n",
```

Comment:
I don't think this is the expected input format. could you test it out and fix this?